test/build.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-env mocha */
+'use strict'
+
+const { expect } = require('../utils/chai')
+const execa = require('execa')
+const process = require('process')
+const { copy, existsSync, readJson } = require('fs-extra')
+const { join } = require('path')
+const bin = require.resolve('../')
+const tempy = require('tempy')
+
+describe('build', () => {
+  if (process.platform !== 'win32') {
```

Comment:
`ipjs` is not supporting windows, trying to figure out why

Reply:
https://github.com/mikeal/ipjs/issues/15 I can't simulate locally nor with a CI branch, so can we just run build on linux+mac while this gets figured out?

Reply:
This will stop us running tests on windows though, right? Or can we run tests under pure ESM instead?

Reply:
well, this just makes us not test that `ipjs` is doing it is thing and building ESM to a dist with `esm` and `cjs`, as the only available test is that one.
It will currently not work on windows and that is a concern to have